dataprocess/dataprocess.go

```diff
@@ -0,0 +1,75 @@
+// Package dataprocess manages code related to the data-accepting process
+package dataprocess
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"image"
+	"image/png"
+	"os"
+
+	"github.com/pkg/errors"
+	pc "go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/rimage"
+	rdkutils "go.viam.com/rdk/utils"
+)
+
+func convertImageToPNG(ctx context.Context, img image.Image) ([]byte, error) {
```

Comment:
We could consider moving convertImageToPNG and WriteImageToPNGFile into the viam-orb-slam3 repo and WritePCDToFile to the viam-cartographer repo, since they will only be used by those modules. In the full modularization project, we'll get rid of the data process for cartographer entirely. But I also think this is fine, and we can change this in full modularization if we want to.

Reply:
I don't think it's worth the effort to move it now since it'll be deleted soon, anyways.